### PR TITLE
2168

### PR DESCRIPTION
**Pull request by Lloyd-K**
Creating a design history for work on allowing automated email scheduling in the admin console.

### CODE REVIEW

File: app/posts/manage-training/2023-12-06-email-scheduling-in-the-admin-console.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+title: Email scheduling in the admin console
```
Comment (frankieroberto) [inline app/posts/manage-training/2023-12-06-email-scheduling-in-the-admin-console.md:2]: I'm never sure what to call the admin bit, but maybe we should avoid using "console" if that means something different to devs? 🤔 

```suggestion
title: Email scheduling in the admin interface
```

File: app/posts/manage-training/2023-12-06-email-scheduling-in-the-admin-console.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+* review the user groups who will receive the emails
+* estimate the number of recipients for each email
+* postpone or delete email sends if circumstances change for schools
+* view how many emails were successfully sent each time
+
+The new dashboard covers all of these needs.
+
+### View
+
+![alt text](screencapture-manage-training-for-ects-5c9ed99fc0a8-herokuapp-admin-performance-emails-2023-12-06-09_18_00.png)
```
Comment (frankieroberto) [inline app/posts/manage-training/2023-12-06-email-scheduling-in-the-admin-console.md:68]: Reminder to add a bit of alt text here.

File: app/posts/manage-training/2023-12-06-email-scheduling-in-the-admin-console.md (hunk 1 above)
Comment (Lloyd-K) [reply app/posts/manage-training/2023-12-06-email-scheduling-in-the-admin-console.md:2]: We definitely need to come up with a better name, but this seems to be the most used for now.

**Review by Lloyd-K — COMMENTED** [review]
Thanks for the review @frankieroberto - if the changes look good to you would you mind approving the PR please? Thanks

**Comment by Lloyd-K** [comment]
Publishing a post on how we designed the new email scheduling screens in the admin console.

**Pull request merged**